shared/components/empty-content/README.md

```diff
@@ -44,6 +43,7 @@ render: function() {
 * <strong>actionURL</strong> — (optional) `href` value for the primary action button.
 * <strong>actionCallback</strong> — (optional) `onClick` value for the primary action button.
 * <strong>actionTarget</strong> - (optional) If ommitted, no target attribute is specified.
+* <strong>featureExample</strong> - (optional) If you initialize the component with this property, containing jsx, it will be added in the bottom of the Empty Content component as an example of what the user could be viewing if there were any content.
```

Comment:
A prop that wraps the given prop with `<FeatureExample />` feels a little generic. What do you think about extending `<EmptyContent />` to include `this.props.children` in its render() (if present) instead?